### ISSUE TITLE
vqgan training and validation loss 

### ISSUE THREAD

**Issue by Daksitha** [evonneng/learning2listen#7]
<img width="1255" alt="Screenshot 2022-10-13 at 11 24 46" src="https://user-images.githubusercontent.com/7329816/195558424-b5749692-02db-416c-8ebb-93d75bb49970.png">

Hi @evonneng, I am training the vqgan with given trevor data and my own data, is this a usual behaviour of the loss curve?

**Comment by evonneng** [comment]
Thank you for the question. Actually yes this is the expected behavior. Usually with the vq side of training, the loss increases before it decreases and saturates. If you’re changing the hyperparameters I find that actually a smaller embedding size and network size with a weight decay schedule usually results in more stable training.
